docs/user/accessing-dashboard/1.6.x-and-below.md

```diff
@@ -1,6 +1,90 @@
 # Accessing Dashboard 1.6.x and below
 
-___It will be moved soon, for now you can [visit Wiki](https://github.com/kubernetes/dashboard/wiki/Accessing-Dashboard---1.6.X-and-below).___
+## `kubectl proxy`
```

Comment:
@floreks Should we describe port forwarding too?

Reply:
Ye, that might be a good idea.

Reply:
Was `kubectl port-forward` implemented in 1.6.X and below? Or is the descriptions for `kubectl port-forward` enough for only 1.7.X and above?

Reply:
>1.7.x is totally fine. We will remove the <1.7.x docs at some point anyway. 1.6 version is not recommended to use with new K8s versions.

Reply:
I see. I added descriptions about `kubectl port-forward` into `>1.7.x` page, please continue to review it. :wink: